packages/cfd/src/Stores/Modules/CFD/cfd-store.js

```diff
@@ -134,6 +142,15 @@ export default class CFDStore extends BaseStore {
                 }
             }
         );
+
+        reaction(
```

Comment:
@hirad-deriv Isn't it against our FE roadmap to add to the stores?
Can we define Hooks or at least Mobx autorun to handle the reactions?

cc: @farzin-deriv

Reply:
@matin-deriv I believe I commented on some other PR before 🤔 Not sure what happened 👀

Reply:
These changes have been added more than 3 months ago. We need to do improvements for this and since we need to meet the deadline of cTrader release, I am not sure if doing so would be possible. We can take care of these issues in our refactoring tasks. What do you say ?
@matin-deriv

Reply:
@markwylde-deriv suggestions would be appreciated 🙏🏼

Reply:
Yes, we should not be extending or adding to the stores. New and changed functionality should be done through hooks.

Is there any reason it would take a lot of work to create a hook out of this?